### DISCUSSION TITLE
Can request advice for specific issues in my own projects here?

### DISCUSSION THREAD

**Discussion by buhtz** [github/securitylab#787] [Q&A] [answered]
The concept of "securitylab" is new to me.

Can request advice for specific issues in my own projects here? Is this allowed or intended to work that way?

As a voluntarily FOSS developer I'm not an expert in all aspects of my own software. If possible security issues are coming up I would like to ask some experts about it. Might be here the correct location for questions like this?

**Comment by xcorail** [comment, marked as answer]
Hey @buhtz 

Of course you can request advice. We provide office hours to the community. To register, please see [this discussion](https://github.com/github/securitylab/discussions/558).

If you need an urgent answer to a specific concern, just [email us](mailto:securitylab@github.com)

Talk to you soon!
